osu.Game/Overlays/BeatmapSet/Scores/ScoreTableLine.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Overlays.BeatmapSet.Scores
+{
+    public class ScoreTableLine : GridContainer
```

Comment:
I don't like how every row in the table is actually a grid container itself. Feels awkward.

Reply:
"Feels awkward" is not a good review. Please suggest an alternative instead. Just from reading the code I see it's done with a purpose, does the design feel awkward or what?

Reply:
It's a table visually but each row is its own grid container. I would prefer a grid container was used to represent the *table*, instead.

See other comments for other weirdness from this design pattern choice.